addon/components/freestyle-menu.js

```diff
@@ -8,5 +8,6 @@ export default Component.extend({
   tagName: '',
 
   emberFreestyle: service(),
+  menuIncludesAllItem: alias('emberFreestyle.menuIncludesAllItem'),
```

Comment:
Best to use `readOnly` instead of `alias` here. `alias` is for when you need to support reading and writing, but here we just need to read this property off the service, and the people setting the property would do so on the service directly.